CIME/SystemTests/err.py

```diff
@@ -39,9 +39,9 @@ def _case_two_custom_prerun_action(self):
         rest_root = os.path.abspath(os.path.join(dout_s_root, "rest"))
         restart_list = ls_sorted_by_fname(rest_root)
         rest_cnt = len(restart_list)
-        expect(rest_cnt >= 1, "No restart files found in {}".format(rest_root))
-        rest_dir = restart_list[max(1, rest_cnt // 2)]
+        rest_dir = restart_list[max(0, rest_cnt // 2)]
```

Comment:
This makes sense to make sure it at least correctly gets the one restart directory if that's all that exists.